src/components/DailyEntries/Entry/styled-components.js

```diff
@@ -15,10 +66,12 @@ export const EntryRoot = styled.div`
 
 export const StartTime = styled.div`
   color: ${text.color};
+  user-select: none;
```

Comment:
Don't repeat yourself here. Do this on parent.

Reply:
Added to DailyEntriesRoot